content/automate/ManageIQ/Cloud/VM/Retirement/Email.class/__class__.yaml

```diff
@@ -3,8 +3,8 @@ object_type: class
 version: 1.0
 object:
   attributes:
-    description: 
-    display_name: 
+    description: Deprecated please use SystemNotificationEmail
```

Comment:
@billfitzgerald0120 
Shouldn't these have slashes System/Notification/Email?

Reply:
@mkanoor  I will update to include slashes.